content/en/docs/tasks/security/health-check/index.md

```diff
@@ -26,12 +26,7 @@ this feature is not needed if the production setup is not using the
 
 To complete this task, you can install Istio using one of the following paths:
 
-* To setup Istio without using Helm, follow the instructions in the
-  [Kubernetes installation guide](/docs/setup/install/kubernetes/). Remember to enable global mutual TLS with:
-
-    {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
-    {{< /text >}}
+* Follow the [installation instructions](/docs/setup/install/kubernetes/#installation-steps) to install Istio.
```

Comment:
This doesn't make sense. It tells users to install Istio without helm, and then the next bullet says "use Helm ..."

Reply:
I think we can just remove this line, the next one explains how to set up with mtls?

Reply:
Yep, I agree, but the next line should then have a link to the appropriate instructions that explains how to do it, but I don't think there are any such clear instructions.

The other issue is that there seem to be a lot of places in the doc (including blogs) that need to be similarly tweaked.

Reply:
> without

@frankbu I got a little confusion. the original doc provides 2 ways to install istio. one is without helm and the other is with helm.

Reply:
Agree with that just provide a single way to use helm to install istio with mtls enabled.